src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/inventory/inventory-devices/inventory-devices.component.ts

```diff
@@ -167,6 +171,14 @@ export class InventoryDevicesComponent implements OnInit, OnDestroy {
       if (col) {
         col.filterable = true;
       }
+
+      if (col?.prop === 'human_readable_type' && this.diskType === 'ssd') {
```

Comment:
@nizamial09 shared devices is supported for disk type with `NVMe` as well, so you have to check that condition as well.

Reply:
we are getting the value of the diskType from the backend (cephadm) and over there ssd will include NVMes also.